app/policies/task_policy.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+class TaskPolicy < ApplicationPolicy
+  def task
+    @record
+  end
+
+  def user_required?
+    false
+  end
+  private :user_required?
```

Comment:
I would probably just make a `private` block as usual, even if it just contains this method.